modules/appeals_api/spec/fixtures/v2/valid_200995.json

```diff
@@ -17,6 +17,20 @@
         "email": "josie@example.com",
         "timezone": "America/Chicago"
       },
+      "claimant": {
```

Comment:
We may only want to put claimant in the extra fixture. I only say this because I recall that recently @cilestin made some changes to HLR and tests claimant only in the extra fixture. Do we want to still do that even though it reduces the claimant data testing coverage?